internal/ast/ast.go

```diff
@@ -250,19 +250,36 @@ type (
 		Body *BlockStmt // body of if branch
 		Else Stmt       // else branch; or nil
 	}
+
+	TryStmt struct {
+		Position
+		Body        *BlockStmt
+		CatchClause []*CatchClause
+		Finalizer   *BlockStmt
+	}
+
+	CatchClause struct {
+		Position
+		Parameter *Ident
+		Body      *BlockStmt
+	}
```

Comment:
I think that would be better a documentation just clarifying that is maybe already claer.
```go
// TryStmt node represents a try catch finaly block.
type TryStmt struct {
	Position
	Body        *BlockStmt // Body of try.
	CatchClause []*CatchClause // List of catches or nil.
	Finalizer   *BlockStmt // Block of finaly statment or nil.
}

// CatchClause node represents a catch inside a try catch statement.
type CatchClause struct {
	Position
	Parameter *Ident // Parameter inside catch block.
	Body      *BlockStmt // Body of catch clause.
}
```

Reply:
Done!